### PR TITLE
Add com.netxms.NetXMSClient51

### PR DESCRIPTION
**Pull request by alkk**
This package is a rich client for network monitoring system NetXMS. Client is Java (SWT) based and depends on Java17+. Package is build from release version which we publish on our site.

This particular package is for 5.1.x branch. We keep protocol compatibility between server and client across major.minor releases, but some users need to access different versions of the server at the same time (e.g. during test deployment/upgrade).

### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly. 
- [X] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. **Link:** https://netxms.com/about-us, right side of the page

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: com.netxms.NetXMSClient51.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+app-id: com.netxms.NetXMSClient51
+runtime: org.freedesktop.Sdk
+runtime-version: "23.08"
```
Comment (bbhtt) [inline com.netxms.NetXMSClient51.yaml:3]: 24.08 is latest.

**Comment by hfiguiere** [comment]
bot, build com.netxms.NetXMSClient51

**Comment by bbhtt** [comment]
bot, build com.netxms.NetXMSClient51

**Pull request closed without merging**